src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLogger.cs

```diff
@@ -39,12 +25,16 @@ internal ConsoleLogger(string name, ConsoleLoggerProcessor loggerProcessor)
 
             _name = name;
             _queueProcessor = loggerProcessor;
+            ScopeProvider = new LoggerExternalScopeProvider();
```

Comment:
Why is this being created here? It has to be passed from the factory

Reply:
I'll remove this. Thanks